pkg/interfaces/contracts/liquidity-mining/IStakelessGauge.sol

```diff
@@ -17,7 +17,20 @@ pragma solidity >=0.7.0 <0.9.0;
 import "./ILiquidityGauge.sol";
 
 interface IStakelessGauge is ILiquidityGauge {
+    /**
+     * @dev Performs a checkpoint, computing how much rewards should be minted for the gauge.
+     */
```

Comment:
```suggestion
     /// @dev Performs a checkpoint, computing how much should be minted for the gauge.
```
Do we want to start using our one-liner comment convention? Same for others. (Also much vs. many grammar nit.)